msamp/operators/fp8_op/fp8_op.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Fp8Op module."""
+
+import ctypes
+
+
+class Fp8Op:
```

Comment:
pls keep consistent with other fp8 class names

Reply:
Sure. Changed class name to FP8Op.